### ISSUE TITLE
SpinnerDialog not working in windows ionic app

### ISSUE THREAD

**Issue by AgNm** [Paldom/SpinnerDialog#43]
I have created an ionic app for android, ios & windows. Spinner Dialog is working for android & ios but it is not working for windows app. I have used exactly same code for all the 3 apps. Can anyone please help.

**Comment by gabrielbazan7** [comment]
this plugin do not support windows 10 platform

**Comment by greybax** [comment]
@AgNm you can use this fork https://github.com/greybax/cordova-plugin-native-spinner which supports win10 instead of it.

**Comment by numiazi** [comment]
i have configure in config file for phonegap build, but i don't understand that, where i do keep show and hide code.. anybody tell me...

**Comment by AgNm** [comment]
@greybax does this plugin   https://github.com/greybax/cordova-plugin-native-spinner supports windows desktop? I used **SpinnerDialog.show();** but it is not working. Please help.

**Comment by greybax** [comment]
@AgNm yes. https://github.com/greybax/cordova-plugin-native-spinner supports windows 10. You can see signature for `show` function [here](https://github.com/greybax/cordova-plugin-native-spinner/blob/master/src/windows/SpinnerDialogProxy.js#L8)

**Comment by tryadelion** [comment]
Could this be made compatible with windows 10 / UWP? that other plugin does not work with Ionic native's V3.
